### PR TITLE
add Flair indexing

### PR DESCRIPTION
**Pull request by kasrakhosravi**
**Description**

Add Flair as a data indexing solution for OP

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

File: community/tools/data-indexing.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Data Indexing
+
+Blockchain data is not query-able by default. Indexing tools help with extracting data from blockchain node, transforming it in a way that is understandable by human or computers, and loading it to a database or any other destination for querying.
```
Comment (sbvegan) [inline community/tools/data-indexing.md:3]: nit, but please update the first sentence to: Blockchain data is not easily queryable by default.

File: community/tools/data-indexing.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Data Indexing
```
Comment (sbvegan) [inline community/tools/data-indexing.md:1]: Lets call this Blockchain Indexers and change the filename to blockchain-indexers.md

**Review by sbvegan — CHANGES_REQUESTED** [review]
Thanks for your contribution! A couple small updates and we can merge this in!

File: community/tools/data-indexing.md (hunk 1 above)
Comment (kasrakhosravi) [reply community/tools/data-indexing.md]: fixed :)

File: community/tools/data-indexing.md (hunk 2 above)
Comment (kasrakhosravi) [reply community/tools/data-indexing.md]: fixed :)

**Review by sbvegan — APPROVED** [review]
Looks good, thanks for your contribution!

**Pull request merged**